tasks.yaml

```diff
@@ -102,16 +102,25 @@ tasks:
     actions:
       - description: Build identity config image
         cmd: docker build --build-arg CA_ZIP_URL="test/cypress/certs/authorized_certs.zip" --build-arg REALM_FILE="test/cypress/realm.json" -t uds-core-config:keycloak --no-cache src
-
       - task: clone-core
-
-      - description: Copy cacert value into uds-core istio gateways
-        task: uds-core-gateway-cacert
-
+      - task: uds-core-gateway-cacert
       - task: build-deploy-custom-slim
+      - task: grafana-group-auth-build-deploy
+      - task: cypress-tests
 
-      - description: Run integration cypress tests
-        task: cypress-tests
+  - name: grafana-group-auth-build-deploy
+    description: Configure existing uds-core/grafana package for group authz, build, deploy grafana
+    actions:
+      - cmd: |
+          sed -i '/^\s*sso:/,/^\s*-\s*name:/ { 
+            /groups:/d
+            /^\s*-\s*name:/a \ \ \ \ \ \ groups:\n\ \ \ \ \ \ \ \ anyOf:\n\ \ \ \ \ \ \ \ \ - \/UDS Core\/Admin
+          }' uds-core/src/grafana/chart/templates/uds-package.yaml
+
+      - cmd: |
+          cd uds-core
+          UDS_PKG=grafana uds run create-single-package
+          uds zarf package deploy build/zarf-package-uds-core-grafana-amd64.tar.zst --confirm --no-progress
```

Comment:
Not to make us wait on another release, but we had briefly discussed maybe putting this in uds-core by default. Does that still seem like a possible route to clean this up (could be done after this PR)?

My only fear would be that someone could use overrides to change their grafana/neuvector group mappings and we would be restricting the client access at a different layer. We could make the group list templated with values as well, but they would still need to override two spots in that case.

Reply:
yeah we've talked about this a few times and i don't think we've actually attempted to try it. I think it would be a good idea to look into this.

Reply:
I'll leave that for a follow-on for sure, but worth exploring. That override scenario is the only thing that gives me pause, but we might be able to handle nicely or just enforce "you must use our groups".

Reply:
https://github.com/defenseunicorns/uds-identity-config/issues/137